packages/ui/src/components/SelectableCard/index.tsx

```diff
@@ -132,6 +165,11 @@ export type SelectableCardProps = {
   tooltip?: string
   label?: ReactNode
   'data-testid'?: string
+  productIcon?: PascalToCamelCaseWithoutSuffix<
```

Comment:
you can't have both props in the same time right ?

Reply:
you can, but you shouldn't

Reply:
you can enforce the type by saying it's one or another